starter_code/js/app.js

```diff
@@ -0,0 +1,25 @@
+$( document ).ready(function(e) {
+
+	$('.readmore').click(function(e){
+		$('.readmore').hide();
+		$('#show-this-on-click').slideDown();
+		$('.readless').removeClass('hide');
+		e.preventDefault();
+	});
+
+	$('.readless').click(function(e){
+		$('#show-this-on-click').slideUp();
```

Comment:
You can also do
```
e.preventDefault();
$('#show-this-on-click').slideUp(function(){
    $('.readless').hide();
    $('.readmore').show();
});
```

The function inside the slideUp will execute after your text has finished sliding up, which is a much nicer user experience.